sksparse/test_cholmod.py

```diff
@@ -275,3 +278,9 @@ def test_CholmodNotPositiveDefiniteError():
     A = -sparse.eye(4).tocsc()
     f = cholesky(A)
     assert_raises(CholmodNotPositiveDefiniteError, f.L)
+
+def test_NaturalOrderingMethod():
```

Comment:
Should probably use underscores instead of camelcase (the one above is an actual class name).